docs/src/main/sphinx/routines/examples.md

```diff
@@ -370,6 +370,56 @@ BEGIN
 END
 ```
 
+## Optional parameter example
+
+Routines can declare usage of other routines as well as other functions. The
```

Comment:
I'm not sure what the first sentence means. Did you mean "Routines can invoke other routines or functions?"

Maybe add another sentence

> You can declare multiple routines with the same name, but that have a different number of arguments or different argument types.

Reply:
Updated